diff --git a/node_modules/@hashgraph/cryptography/lib/Ed25519PrivateKey.cjs b/node_modules/@hashgraph/cryptography/lib/Ed25519PrivateKey.cjs
index 527800e..5df20a7 100644
--- a/node_modules/@hashgraph/cryptography/lib/Ed25519PrivateKey.cjs
+++ b/node_modules/@hashgraph/cryptography/lib/Ed25519PrivateKey.cjs
@@ -10,7 +10,7 @@ var _tweetnacl = _interopRequireDefault(require("tweetnacl"));
 var hex = _interopRequireWildcard(require("./encoding/hex.cjs"));
 var random = _interopRequireWildcard(require("./primitive/random.cjs"));
 var slip10 = _interopRequireWildcard(require("./primitive/slip10.cjs"));
-var _nodeForge = _interopRequireDefault(require("node-forge"));
+// var _nodeForge = _interopRequireDefault(require("node-forge"));
 function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
 function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && Object.prototype.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
diff --git a/node_modules/@hashgraph/cryptography/lib/Ed25519PublicKey.cjs b/node_modules/@hashgraph/cryptography/lib/Ed25519PublicKey.cjs
index 28de3df..3471fff 100644
--- a/node_modules/@hashgraph/cryptography/lib/Ed25519PublicKey.cjs
+++ b/node_modules/@hashgraph/cryptography/lib/Ed25519PublicKey.cjs
@@ -9,7 +9,7 @@ var _BadKeyError = _interopRequireDefault(require("./BadKeyError.cjs"));
 var _tweetnacl = _interopRequireDefault(require("tweetnacl"));
 var _array = require("./util/array.cjs");
 var hex = _interopRequireWildcard(require("./encoding/hex.cjs"));
-var _nodeForge = _interopRequireDefault(require("node-forge"));
+// var _nodeForge = _interopRequireDefault(require("node-forge"));
 function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
 function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && Object.prototype.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
diff --git a/node_modules/@hashgraph/cryptography/lib/encoding/pem.cjs b/node_modules/@hashgraph/cryptography/lib/encoding/pem.cjs
index cc35f08..845e9f8 100644
--- a/node_modules/@hashgraph/cryptography/lib/encoding/pem.cjs
+++ b/node_modules/@hashgraph/cryptography/lib/encoding/pem.cjs
@@ -13,7 +13,7 @@ var base64 = _interopRequireWildcard(require("./base64.cjs"));
 var _Ed25519PrivateKey = _interopRequireDefault(require("../Ed25519PrivateKey.cjs"));
 var _EcdsaPrivateKey = _interopRequireDefault(require("../EcdsaPrivateKey.cjs"));
 var asn1 = _interopRequireWildcard(require("asn1js"));
-var _nodeForge = _interopRequireDefault(require("node-forge"));
+// var _nodeForge = _interopRequireDefault(require("node-forge"));
 var hex = _interopRequireWildcard(require("./hex.cjs"));
 var aes = _interopRequireWildcard(require("../primitive/aes.cjs"));
 var _buffer = require("buffer");
diff --git a/node_modules/@hashgraph/cryptography/src/Ed25519PrivateKey.js b/node_modules/@hashgraph/cryptography/src/Ed25519PrivateKey.js
index 3fc6f27..570df22 100644
--- a/node_modules/@hashgraph/cryptography/src/Ed25519PrivateKey.js
+++ b/node_modules/@hashgraph/cryptography/src/Ed25519PrivateKey.js
@@ -4,7 +4,7 @@ import nacl from "tweetnacl";
 import * as hex from "./encoding/hex.js";
 import * as random from "./primitive/random.js";
 import * as slip10 from "./primitive/slip10.js";
-import forge from "node-forge";
+// import forge from "node-forge";
 
 export const derPrefix = "302e020100300506032b657004220420";
 export const derPrefixBytes = hex.decode(derPrefix);
diff --git a/node_modules/@hashgraph/cryptography/src/Ed25519PublicKey.js b/node_modules/@hashgraph/cryptography/src/Ed25519PublicKey.js
index 1e2a2cc..9e1c804 100644
--- a/node_modules/@hashgraph/cryptography/src/Ed25519PublicKey.js
+++ b/node_modules/@hashgraph/cryptography/src/Ed25519PublicKey.js
@@ -3,7 +3,7 @@ import BadKeyError from "./BadKeyError.js";
 import nacl from "tweetnacl";
 import { arrayEqual } from "./util/array.js";
 import * as hex from "./encoding/hex.js";
-import forge from "node-forge";
+// import forge from "node-forge";
 
 const derPrefix = "302a300506032b6570032100";
 const derPrefixBytes = hex.decode(derPrefix);
diff --git a/node_modules/@hashgraph/cryptography/src/encoding/pem.js b/node_modules/@hashgraph/cryptography/src/encoding/pem.js
index e4084a2..2f5a319 100644
--- a/node_modules/@hashgraph/cryptography/src/encoding/pem.js
+++ b/node_modules/@hashgraph/cryptography/src/encoding/pem.js
@@ -5,7 +5,7 @@ import * as base64 from "./base64.js";
 import Ed25519PrivateKey from "../Ed25519PrivateKey.js";
 import EcdsaPrivateKey from "../EcdsaPrivateKey.js";
 import * as asn1 from "asn1js";
-import forge from "node-forge";
+// import forge from "node-forge";
 import * as hex from "./hex.js";
 import * as aes from "../primitive/aes.js";
 import { Buffer } from "buffer";
