diff --git a/node_modules/react-native-screenshot-prevent/ios/RNScreenshotPrevent.m b/node_modules/react-native-screenshot-prevent/ios/RNScreenshotPrevent.m
index 9cddcc7..78d91ec 100644
--- a/node_modules/react-native-screenshot-prevent/ios/RNScreenshotPrevent.m
+++ b/node_modules/react-native-screenshot-prevent/ios/RNScreenshotPrevent.m
@@ -110,7 +110,7 @@ -(void) addSecureTextFieldToView:(UIView *) view {
     [view sendSubviewToBack:secureField];
     [view addSubview:secureField];
     [view.layer.superlayer addSublayer:secureField.layer];
-    [[secureField.layer.sublayers objectAtIndex:0] addSublayer:view.layer];
+    [[secureField.layer.sublayers lastObject] addSublayer:view.layer];
 }
 
 // TODO: not working now, fix crash on _UITextFieldCanvasView contenttViewInvalidated: unrecognized selector sent to instance
